### DISCUSSION TITLE
Add ai features

### DISCUSSION THREAD

**Discussion by ivehgr** [Rem0o/FanControl.Releases#2867] [Ideas]
Maybe you could add AI feature for the fan curves to have the optimal temperature speed, temperature noise or temperature performance ratio. Further, an automatic sensor naming and a recommendation for temperature sensor.

**Comment by Rem0o** [comment]
I'm not really into buzzwords, but I do know quite a bit on that subject from past work. 

I haven't seen a realistic/useful use case for AI in fancontrol. Any meaningful AI feature that I can think of would be a huge project in it of itself, and I don't see how that would fit in a small/to-the-point program like FanControl.  

AI can't auto-name the fans if it can't "see" your system, where you plugged what.  Automatic-configuring, well that would be the most "interesting" subject, but again, the cost-benefit doesn't add up for me.

**Comment by ivehgr** [comment]
Why is ai not useful. It makes a hell of work to configure all curves.

Rémi Mercier ***@***.***> schrieb am Mi. 11. Dez. 2024 um
05:07:
